apps/ledger-live-mobile/src/screens/SyncOnboarding/SoftwareChecksStep.tsx

```diff
@@ -210,12 +203,14 @@ const SoftwareChecksStep = ({ device, isDisplayed, onComplete }: Props) => {
         } else if (devicePermissionState === "refused") {
           nextDrawerToDisplay = "cancelled";
         }
+      } else if (genuineCheckStatus === "failed") {
+        nextDrawerToDisplay = "cancelled";
```

Comment:
Should we rename the value `"cancelled"` to `"failed"` in `GenuineCheckUiDrawerStatus` ? I don't think `cancelled` makes sense anymore - the other time this value is used is just above on:

```
  } else if (devicePermissionState === "refused") {
          nextDrawerToDisplay = "cancelled";
        }
```

and if it was `"failed"` it would still make sense